Appraisals

```diff
@@ -369,6 +369,7 @@ elsif Gem::Version.new('2.2.0') <= Gem::Version.new(RUBY_VERSION) \
       gem 'sqlite3', '~> 1.3.6'
       gem 'sucker_punch'
       gem 'typhoeus'
+      gem 'que', '1.0.0.beta4'
```

Comment:
```suggestion
      gem 'que', '>= 1.0.0.beta'
```

If we set it to `>=` we'll automatically install (and test with) the stable version `1.0` when it's released.
I'd also suggest you try with the lowest version that's compatible, which is likely `1.0.0.beta` in our case.
If this PR works for all beta releases, we don't have a strong reason to only support `beta4`.

That being said, if you see that the difference between the beta releases is significant, then go ahead and lock it to a higher release number.

Reply:
Thank you for the tip.

We can only go as low as `1.0.0.beta2` since that's when the `middleware` functionality becomes available. I think we should leave that version as a hard requirement.